docs/core.md

```diff
@@ -69,16 +69,16 @@ A working example: https://codesandbox.io/s/jotai-r3f-fri9d
 
 # How atom dependency works
 
-To begin with, let's explain this. In the current implementation, every time we invoke the "read" function, we refresh dependents.
+To begin with, let's explain this. In the current implementation, every time we invoke the "read" function, we refresh dependencies.
```

Comment:
If A depends on B, B is a dependency of A and A is a dependent of B. How does that sound?

Reply:
Added this in 👍